datasimulator/submit_data_utils.py

```diff
@@ -10,6 +10,8 @@
 
 def submit_test_data(host, project, dir, access_token_file, max_chunk_size=1):
 
+    api_root_endpoint = host + "/api/v0/submission"
+    api_program_endpoint = host + "/api/v0/submission/" + project.split("/")[0]
     api_endpoint = host + "/api/v0/submission/" + project
```

Comment:
```
program_name = project.split("/")[0]
api_program_endpoint = "{}/{}".format(api_root_endpoint, program_name)
api_endpoint = "{}/{}".format(api_root_endpoint, project)
```
you could use this, and then use `program_name` directly when generating the program json
also i think a docstring for this function to mention that `project` is in the format "program/project" would be nice